### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -1172,7 +1172,6 @@
       "placeholder": "Enter hexadecimal data"
     },
     "error": {
-      "feeTooLow": "fee too low",
```
Comment (thisconnect) [inline frontends/web/src/locales/en/app.json:1175]: why is feeTooLow gone again?

Comment (benma) [reply frontends/web/src/locales/en/app.json:1175]: hmm good question, i guess we pushed locize from master while this was in the staging branch :/ will fix

Comment (benma) [reply frontends/web/src/locales/en/app.json:1175]: First: https://github.com/digitalbitbox/bitbox-wallet-app/pull/987

**Comment by benma** [comment]
Will re-open on `master` later.

**Pull request closed without merging**